src/main/java/seedu/address/model/Entity/Score.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.Entity;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Score {
+
+    private static final String MESSAGE_CONSTRAINTS = "Score should contain only positive integers from 0 to 100";
+    private int score;
+
+
+
+    /**
+     * Constructs a {@code Score}
+     *
+     * @param score A valid score of a team.
+     */
+    public Score(int score) {
+        requireNonNull(score);
+        checkArgument(isValidScore(score), MESSAGE_CONSTRAINTS);
+        this.score = score;
+    }
+
+    /**
+     * Returns if a given string is a valid score.
+     *
+     * @param score Score.
+     * @return boolean whether score is in valid score format.
+     */
+    public static boolean isValidScore(int score) {
+        return score >= 0 && score <= 100;
+    }
+
+
+    /**
+     * Returns string representation of object.
+     *
+     * @return Score in string format.
+     */
+   public String toString(){
+    return Integer.toString(this.score);
+   }
+
+    /**
+     * Returns string representation of object, for storage.
+     *
+     * @return Score in string format.
+     */
+   public String toStorageValue(){
```

Comment:
toStorageValue should return the parameter that is used in the constructor of the Score object. In this case, this should be returning this.score (the int)